core/server/helpers/asset.js

```diff
@@ -1,14 +1,25 @@
 // # Asset helper
-// Usage: `{{asset "css/screen.css"}}`, `{{asset "css/screen.css" ghost="true"}}`
+// Usage: `{{asset "css/screen.css"}}`, `{{asset "css/screen.css" ghost="true"}}`,
+// `{{asset "locales/{lang}.css" lang=(lang)}}`
```

Comment:
What's the purpose of the changes here?
To generate a asset url to the theme language files?

FYI: No css translations for the first version, [see](https://github.com/TryGhost/Ghost/pull/8437#issuecomment-330309612).

Reply:
The purpose of the changes in the `{{asset}}` helper is to allow nested helpers (Handlebars subexpressions). If approved by the team, this would be initially used to improve caching and path management for css overrides (just for some themes with visible text as css content, e.g. a couple of text lines in those themes based on the blog design of Casper 1.4). It could also be potentially used for any asset management.

What we call css content translation in fact does not use our i18n translation function, but a normal css override. That's something optional to be implemented by theme designers for those special themes. So, of course, css override is not a feature we can remove, we can only allow to improve it with `{{asset}}` or not.

Although I would prefer to keep the new nested helper capabilities of the `{{asset}}` helper, removing them probably would just cause some caching and path inconveniences for css content overrides implemented by theme designers in those special themes that need the overriding for some texts. So, naturally, you can say me what to do.

Reply:
I appreciate taking time to explain all of this, but can we please have the first i18n basic version without any css features. We can add and discuss anything on top. We can raise issues for each feature and design how it will be implemented. I don't want to merge something, which we might remove next year, because it will then work differently.

Reply:
Done.